spec/menu_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'menu'
+
+describe Menu do
+
+  subject(:menu) { Menu.new }
+
+  it "can have dishes added to it" do
+    menu.add :burger, 9.99
+    menu.add :chips, 1.99
+    menu.add :coke, 0.99
+    expect(menu.show).to eq "Burger - £9.99 // Chips - £1.99 // Coke - £0.99"
```

Comment:
This tests both `add` and `show` -> figure out a way of showing the menu raw data.